dist/webgazer.umd.js

```diff
@@ -1,3 +1,12 @@
+/*!
```

Comment:
This is great too, but what I actually meant is to add the license field to the package.json: https://docs.npmjs.com/files/package.json